### ISSUE TITLE
Add a way to open the popup using a contexrt menu option rather than double-click

### ISSUE THREAD

**Issue by a2sheppy** [krv5vignesh/Dictionary-extension#13]
I use double-click to select things all the time, so the current double-click activation is interfering with that. Could you please add an option to disable that and just use a context menu item on the selection instead?

**Comment by Two-Tone** [comment]
I'd prefer if we had a trigger key, like what the Google Dictionary extension does in Chrome.

**Comment by onlycparra** [comment]
Yes, please. Having to close the dictionary every time I select text is quite annoying.
Just having the option to do `[Shift]+[Double Click]` would make the trick.
